backend/src/auth/midleware/ValidationMidleware.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from "express";
+import { validateOrReject } from 'class-validator';
+
+export function ValidationMidleware(dto: any) {
```

Comment:
dto must be a type of Class (constructor signature;
you can use `type Dto = new () => any`